webpack/ForemanInventoryUpload/ForemanInventoryConstants.js

```diff
@@ -3,3 +3,5 @@ import { translate as __ } from 'foremanReact/common/I18n';
 export const INVENTORY_PAGE_TITLE = __('Red Hat Inventory Uploads');
 
 export const DOCS_BUTTON_TEXT = __(' Documentation');
+
+export const TASKS_HISTORY_BUTTON_TEXT = __(' Tasks history');
```

Comment:
I think `Actions history` is more understandable for the user. Tasks are implementation details here.

Reply:
no problem.. :+1:  I thought about `tasks` from the `tasks page`